config/jest/setup.js

```diff
@@ -1,7 +1,7 @@
 import jsdom from 'jsdom'
 import sinon from 'sinon'
 import Enzyme from 'enzyme'
-import Adapter from 'enzyme-adapter-react-16'
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
```

Comment:
https://github.com/enzymejs/enzyme/issues/2429#issuecomment-679265564 indicates that the Enzyme maintainers won't be releasing support soon, so here's the drop-in replacement.